core/flyout.js

```diff
@@ -704,8 +704,9 @@ Blockly.Flyout.prototype.show = function(xmlList) {
         var isLabel = tagName == 'LABEL';
         var text = xml.getAttribute('text');
         var callbackKey = xml.getAttribute('callbackKey');
+        var className = xml.getAttribute('class');
         var curButton = new Blockly.FlyoutButton(this.workspace_,
-            this.targetWorkspace_, text, callbackKey, isLabel);
+            this.targetWorkspace_, text, callbackKey, isLabel, className);
```

Comment:
a thought: I wonder if we should just pass the xml chunk through to FlyoutButton and let its constructor handle this code instead of reading all the pieces out of it.  Then this else if becomes mostly contents.push( button stuff); gaps.push(gap);